src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -3381,11 +3361,10 @@ Real64 CalcZoneSystemACH(EnergyPlusData &state, int const ZoneNum)
 
 Real64 CalcZoneSupplyAirTemp(EnergyPlusData &state, int const ZoneNum)
 {
-    using namespace DataZoneEquipment;
```

Comment:
There is no reason for keeping these using namespace in these little functions, correct?